react-time-track/src/views/edit-user.js

```diff
@@ -19,12 +18,9 @@ function EditUser({ user_id }) {
     getUser(user_id).then(data => setUser(data));
   }, []);
 
-  function handleChange(e, key) {
-    setUser({ ...user, [key]: e.target.value });
-  }
-
   function handleSubmit(e) {
     e.preventDefault();
+    console.log(user);
```

Comment:
Don't forget to delete those `console.log()`.

Reply:
Thanks, I have forgotten do that.